scripts/dev_cli.sh

```diff
@@ -0,0 +1,293 @@
+}
+
+cmd_build() {
+  build_type="$BUILD_TYPE"
+  runlevel="$RUNLEVEL"
+  while [ $# -gt 0 ]; do
+    case "$1" in
+            "-h"|"--help")  { cmd_help; exit 1; } ;;
+            "--debug")      { build_type="Debug"; } ;;
+            "--release")    { build_type="Release"; } ;;
+            "--runlevel")
+              shift
+              [[ "$1" =~ ^(Debug|Release) ]] || \
+                die "Invalid runlevel: $1. Valid options are \"Debug\" and \"Release\"."
+                runlevel="$1"
+                ;;
+            *)
+              die "Unknown build argument: $1. Please use --help for help."
+              ;;
+	  esac
+	  shift
+  done
+
+  ensure_build_dir
+  # ensure_latest_ctr
+
+  $DOCKER_RUNTIME run \
+    --workdir "$CTR_FUZZUF_ROOT_DIR" \
+    --rm \
+    --volume "$FUZZUF_ROOT_DIR:$CTR_FUZZUF_ROOT_DIR" \
+    "$CTR_IMAGE" \
+    /bin/bash -c "set -eux \
+    && cmake -B $CTR_FUZZUF_BUILD_DIR \
+      -DCMAKE_BUILD_TYPE=$build_type \
+      -DDEFAULT_RUNLEVEL=$runlevel \
+      -DPIN_ROOT=$PIN_ROOT \
+      -DENABLE_DOXYGEN=1 \
+    && cmake --build $CTR_FUZZUF_BUILD_DIR -j$(nproc) \
+    && cmake --build $CTR_FUZZUF_BUILD_DIR --target die"
+
+  fix_dir_perms $?
+}
+
+cmd_clean() {
+  ensure_build_dir
+  # ensure_latest_ctr
+
+  $DOCKER_RUNTIME run \
+    --workdir "$CTR_FUZZUF_BUILD_DIR" \
+    --rm \
+    --volume "$FUZZUF_ROOT_DIR:$CTR_FUZZUF_ROOT_DIR" \
+    "$CTR_IMAGE" \
+    cmake --build $CTR_FUZZUF_BUILD_DIR --target clean
+
+  fix_dir_perms $?
+}
+
+cmd_tests() {
+  ensure_build_dir
+  # ensure_latest_ctr
+
+  $DOCKER_RUNTIME run \
+    --workdir "$CTR_FUZZUF_BUILD_DIR" \
+    --rm \
+    --volume "$FUZZUF_ROOT_DIR:$CTR_FUZZUF_ROOT_DIR" \
+    "$CTR_IMAGE" \
+    cmake --build $CTR_FUZZUF_BUILD_DIR --target test
+
+  fix_dir_perms $?
+}
+
+cmd_shell() {
+  ensure_build_dir
+  ensure_latest_ctr
+
+  say_warn "Starting a privileged shell prompt as root ..."
+  say_warn "WARNING: Your $FUZZUF_ROOT_DIR folder will be bind-mounted in the container under $CTR_FUZZUF_ROOT_DIR"
+
+  $DOCKER_RUNTIME run \
+    -ti \
+    --workdir "$CTR_FUZZUF_ROOT_DIR" \
+    --rm \
+    --volume "$FUZZUF_ROOT_DIR:$CTR_FUZZUF_ROOT_DIR" \
+    --env USER="root" \
+    --entrypoint bash \
+    "$CTR_IMAGE"
+
+  fix_dir_perms $?
+}
+
+cmd_help() {
+    echo ""
+    echo "fuzzuf $(basename $0)"
+    echo "Usage: $(basename $0) <command> [<command args>]"
+    echo ""
+    echo "Available commands:"
+    echo ""
+    echo "    build [--debug|--release] [--runlevel Debug|Release]"
+    echo "        Build the fuzzuf binaries."
+    echo "        --debug               Build the debug binaries. This is the default."
+    echo "        --release             Build the release binaries."
+    echo "        --runlevel            Select default runlevel. Default is Debug."
+    echo ""
+    echo "    tests"
+    echo "        Run the fuzzuf tests."
+    echo ""
+    echo "    build-container"
+    echo "        Build the fuzzuf container."
```

Comment:
```suggestion
    echo "        Build the fuzzuf container locally."
```
Clarify the command will build a container locally.